test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegatePointInstanceSelection.py

```diff
@@ -134,7 +134,7 @@ def _RunTest(self):
         newSelection.clear()
 
         # Select the back half of the instances.
-        for instanceIndex in range(self._numInstances / 2, self._numInstances):
+        for instanceIndex in range(self._numInstances // 2, self._numInstances):
```

Comment:
In py2, dividing two integer s uses integer division, also known as floor division (apply the floor after division). In py3 this was deprecated. So use the floor division operator "//" avail in python 2.2 and later.